README.md

```diff
@@ -39,21 +28,11 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
```

Comment:
- [ ] Please remove this header and other ones in your README file which are empty and have no further description